src/Canvas.tsx

```diff
@@ -197,8 +197,8 @@ function Canvas<R, K extends keyof R>({
           key={key}
           rowIdx={rowIdx}
           row={row}
-          columnMetrics={columnMetrics}
-          viewportColumns={viewportColumns}
+          columnMetrics={columnMetrics as ColumnMetrics<R, never>}
+          viewportColumns={viewportColumns as readonly CalculatedColumn<R, never>[]}
```

Comment:
Here as well we should avoid type assertions as they may mask broken types in the future.